packages/domain/src/use-cases/transactions/SupportedTransactionRequest.ts

```diff
@@ -1,6 +1,6 @@
-import { JustProtocolRequest } from '../../entities';
-import { ClaimHandleRequest } from '../profile';
+import { JustProtocolRequest } from '../../entities/Transactions';
```

Comment:
Is this also for circular dep? There shouldn't be

Reply:
you are right, reverting this change